spec/features/sessions/user_authorization_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Authorization', type: :feature do
+  it 'redirects the user if not logged in' do
```

Comment:
Use context as describe here http://www.betterspecs.org/

The test should have two contexts 'if user is logged in' and 'if user is not logged in'. The 'it' should than only state what we expect e. g. 'it redirects the user', 'it shows a message', it should not explain the context.

Reply:
Yes good catch!

Reply:
Thanks. It really looks better that way.